msteams-platform/concepts/authentication/nested-authentication.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Nested app authentication
+description: Learn about nested authentication in the Microsoft Teams platform.
+ms.date:
+ms.topic: conceptual
+author: v-ypalikila
+ms.author: surbhigupta
+ms.localizationpriority: medium
+---
+
+# Nested app authentication
+
+> [!NOTE]
+>
+> * Nested app authentication is available only in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
+> * Nested app authentication is only supported in tabs.
+
+Nested app authentication (NAA) is a new authentication protocol for single page applications that are embedded in host environments like Teams, Outlook, and Microsoft 365. It simplifies the authentication process and
+Nested app authentication (NAA) is a new authentication protocol for single page applications that are embedded in host environments like Teams, Outlook, and Microsoft 365. It simplifies the authentication process to facilitate single sign-on (SSO) across applications nested within supported host apps and provides several advantages over the On-Behalf-Of (OBO) flow.
+
+* You only need to use the MSAL.js library and don’t need the getAuthToken function in TeamsJS.
+* You can call services such as Microsoft Graph with an access token from your client code as an SPA. There’s no need for a middle-tier server.
+* You can use incremental and dynamic consent for scopes.
+* You don't need to preauthorize your hosts (For example, Teams, Office) to call your endpoints.
+
+The NAA model supports a primary identity that includes multiple app identities. Microsoft utilizes this framework in Office Add-Ins and Teams Tabs and Personal apps.
```

Comment:
Hm. I'm not sure what this is saying in the first sentence

Reply:
Reworked on the content to give better context.